Vision/README.md

```diff
@@ -1,51 +1,47 @@
 # The W3C Vision: An Emphasis on Values
 
 ## Introduction
-The World Wide Web was conceived more than 25 years ago 
+The World Wide Web was originally conceived 
 as a tool for sharing information. 
-It has become much more than that; 
-it is a fundamental part of the lives of much of humanity, 
-enabling access to information, education, 
+It has evolved into a fundamental part of humanity, 
+providing access to knowledge, education, 
 commerce and shopping, social experiences, 
-civic functions, entertainment, and more. 
+civic duties, entertainment, and more.
```

Comment:
```suggestion
government services, entertainment, and more.
```

Reply:
That is less clear, not more, imo.

Reply:
I am literally not sure what civic duties I access - presumably paying taxes except I don't do that through the web. I know a wide range of government services I rely on getting through the web.

How would you translate the term into some other languages? If they are romance or germanic, that would probably help me understand better.

That said, this is editorial, and I am not the editor.

Reply:
FWIW I find "civic functions" clearer than "civic duties".

Reply:
Being able to access government info and support is a function , not a civic duty. Many actions that used to take a trip to an obscure government office in a hard-to-reach location during inconvenient hours can now be performed online, and they aren't duties.

Reply:
reverted.